src/unitxt/loaders.py

```diff
@@ -177,18 +177,13 @@ def verify(self):
                 self._requirements_list.append(requirement)
         super().verify()
 
-    def filtered_load(self, dataset):
+    def filter_load(self, dataset):
         if not settings.allow_unverified_code:
             raise ValueError(
                 f"{self.__class__.__name__} cannot run use filtering_lambda expression without setting unitxt.settings.allow_unverified_code=True or by setting environment variable: UNITXT_ALLOW_UNVERIFIED_CODE."
             )
         logger.info(f"\nLoading filtered by: {self.filtering_lambda};")
-        return MultiStream(
-            {
-                name: dataset[name].filter(eval(self.filtering_lambda))
-                for name in dataset
-            }
-        )
+        return dataset.filter(eval(self.filtering_lambda))
```

Comment:
Should this line be protected by `allow_unverified_code` (https://github.com/IBM/unitxt/blob/6b1d93d2914e2551668c1803f59392411cee05c6/src/unitxt/settings_utils.py#L121)
@yoavkatz  can you advise?

Reply:
Hi @elronbandel , it is well protected, in line 181, a few lines above the line you commented about